pkg/dab/find.go

```diff
@@ -99,6 +99,18 @@ func FindActionableFromFS(
 	m *wfapi.Module, p *wfapi.Plot, f *wfapi.Formula,
 	foundPath string, remainingSearchPath string, err error,
 ) {
+	defer func() {
+		if remainingSearchPath == "." {
+			// canonicalize to an empty string for an empty remaining search path
+			remainingSearchPath = ""
+		}
+	}()
+	if filepath.IsAbs(searchPath) {
+		err = serum.Error(wfapi.ECodeArgument,
+			serum.WithMessageTemplate("search path {{path|q}} may not be an absolute path"),
+			serum.WithDetail("path", searchPath),
+		)
+	}
```

Comment:
I think I'd be just as happy having a panic.  (My thinking is that this function is far enough away from direct user inputs that the path parameters probably should've been munged and checked for good form at least once already somewhere closer to the main method.)

But it's a weakly held opinion.  And would be easy to change later one way or the other.